tests/testfx.lua

```diff
@@ -244,17 +246,19 @@
 -- Some helper functions
 --
 
-	function test.createsolution()
-		local sln = solution "MySolution"
+	function test.createWorkspace()
+		local wrk = workspace("MySolution")
 		configurations { "Debug", "Release" }
 
-		local prj = project "MyProject"
-		language "C++"
-		kind "ConsoleApp"
+		local prj = project("MyProject")
+		language("C++")
+		kind("ConsoleApp")
```

Comment:
Curious change in pattern; are you moving away from api's without brackets in terms of styling?